src/google/protobuf/compiler/objectivec/BUILD.bazel

```diff
@@ -6,6 +6,15 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
 load("//build_defs:cpp_opts.bzl", "COPTS")
 
+cc_library(
+    name = "names",
+    hdrs = ["names.h"],
```

Comment:
This still feels like a breaking change, as grpc/etc. have to update to now import names.h, should we put the private things in names.h and use objectivec_helpers.h as the public one so they have less code to change?

Reply:
In other languages we use `names.h` as the public header exposing naming functions only. I would like to keep this convention consistent between languages.